packages/material-ui/src/NoSsr/NoSsr.js

```diff
@@ -68,10 +65,6 @@ NoSsr.propTypes = {
   fallback: PropTypes.node,
 };
 
-if (process.env.NODE_ENV !== 'production') {
```

Comment:
For some reason, this breaks API generation. 

Error: The "defer"" property is missing a description
    at generateMarkdown.js:238:13

Reply:
This made me find that an extra `"` is added to the error message

Reply:
Check CssBasline.js.

Reply:
It's weird that `['propTypes' + '']` fixes this